Testscripts/Linux/validate-da.sh

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/env bash
+#######################################################################
```

Comment:
In general, if you add LogMsg or LogErr, it helps your future debugging in TestExecution.log and TestExecutionErr.log